### DISCUSSION TITLE
Solids aka 3d geometries

### DISCUSSION THREAD

**Discussion by Applesauce314** [NetTopologySuite/NetTopologySuite#730] [Q&A] [answered]
a cursory examination of this repository and accompanying documentation, seems to indicate that NTS targets2d topology only, not 3D solids. Is this correct? I see there is a Z ordinate on Coordinate, but I am not seeing anything like Volume, or Surface, and the comments on the Z seem to indicate there is not much built in support. Is this a correct assessment?
Don't want to move on because I missed something obvious.

**Comment by FObermaier** [comment, marked as answer]
NetTopologySuite does not handle solids.
The z-ordinate is only for elevation.
